index.go

```diff
@@ -44,9 +44,19 @@ type Index interface {
 	// AddWithIDs is like Add, but stores xids instead of sequential IDs.
 	AddWithIDs(x []float32, xids []int64) error
 
-	// Applicable only to IVF indexes: Return a map of centroid ID --> []vector IDs
-	// for the cluster.
-	ObtainClusterToVecIDsFromIVFIndex() (ids map[int64][]int64, err error)
+	// Returns true if the index is an IVF index.
+	IsIVFIndex() bool
+
+	// CountVectorsPerCluster returns a map where the keys are cluster IDs
```

Comment:
Add a comment that is applicable to IVFIndex only and for consistency rename this to ..
```
ObtainRelevantClusterDocCountsFromIVFIndex(..)
```